src/jobtypes.py

```diff
@@ -111,13 +112,14 @@ def _get_jobnames(self, job, testtype, buildername):
         signature = ''  # TH specific
         buildtype = job['platform_option']
         platform = job['platform']
+        buildplatform = job['build_system_type']
```

Comment:
Why not build_system_type? I fine it better to keep consistent naming of concepts across multiple code bases.